source/unified-test-format/unified-test-format.rst

```diff
@@ -1079,6 +1079,11 @@ The structure of this object is as follows:
   assert that the error does not contain any of the specified labels (e.g. using
   the ``hasErrorLabel`` method).
 
+- ``errorResponse``: Optional document. A value corresponding to the expected
```

Comment:
it seems we're treating this as a root document. should we call that out here or in Evaluating Matches?